packages/next/src/shared/lib/get-img-props.ts

```diff
@@ -86,7 +86,8 @@ export type ImageLoader = (p: ImageLoaderProps) => string
 // built-in loaders, not for a custom loader() prop.
 type ImageLoaderWithConfig = (p: ImageLoaderPropsWithConfig) => string
 
-export type PlaceholderValue = 'blur' | 'empty'
+type DataURL = string & Record<never, never>
```

Comment:
```suggestion
type DataURL = `data:${string}`;
```

Reply:
whoa I didn't know TS allows this. thank you!